lib/Importer/ONNXModelLoader.cpp

```diff
@@ -223,6 +223,73 @@ void glow::setOnnxDefineSymbol(const std::vector<std::string> &strs) {
   onnxDefineSymbol = strs;
 }
 
+ONNX_NAMESPACE::GraphProto glow::parseOnnxFile(const std::string &fileName) {
+  ::ONNX_NAMESPACE::GraphProto graphProto;
+  std::ifstream inputFileStream(fileName, std::ios::in | std::ios::binary);
+  CHECK(inputFileStream) << "Can't find the input file for " << fileName;
+  google::protobuf::io::IstreamInputStream protobufFileStream(&inputFileStream);
+  google::protobuf::io::CodedInputStream codedStream(&protobufFileStream);
+  codedStream.SetTotalBytesLimit(MAX_PROTO_SIZE, MAX_PROTO_SIZE);
+  bool parsedSuccessfully = graphProto.ParseFromCodedStream(&codedStream);
+  CHECK(parsedSuccessfully) << "Failed to parse GraphProto";
+  return graphProto;
+}
+
+void glow::fillPlaceholders(const ONNX_NAMESPACE::GraphProto &inputGroup,
+                            PlaceholderBindings *bindings,
+                            std::vector<Tensor> *partialTensorPayloads,
+                            bool enablePartialTensor) {
+  for (const auto &tensorProto : inputGroup.initializer()) {
+    auto *tensor =
+        bindings->get(bindings->getPlaceholderByName(tensorProto.name()));
+    CHECK(tensor);
+    size_t fullSize = tensor->getSizeInBytes();
+    const auto fullType = tensor->getType();
+    auto error = loadTensor(tensorProto, tensor);
+    bool hasError = ERR_TO_BOOL(std::move(error));
+    CHECK(!hasError) << "Cannot load input tensor";
+    size_t loadedSize = tensor->getSizeInBytes();
+    if (loadedSize != fullSize) {
+      if (enablePartialTensor) {
```

Comment:
small nit, no big deal: seems like we could simply check `if (partialTensorPayloads)` instead here, and get rid of `enablePartialTensor`? In repro we'd just pass `enablePartialTensor ? &partialTensorPayloads : nullptr`.